src/api/app/views/webui2/webui/project/_subprojects_table.html.haml

```diff
@@ -0,0 +1,11 @@
+%h3 #{table_title} of #{project}
```

Comment:
you should pluralize `table_title` here

Reply:
@Ana06  please read the comment from @dmarcoux

Reply:
i will keep it singular as agreed by the others reviewers.

Reply:
I was arguing about the title of the columns in the table. However, @Ana06 is right here. It's different here since it's the header of the page.

Reply:
hm @dmarcoux  ok, because it used to be singular, no?

Reply:
It was a mistake to have the page's header in singular :confused:

Reply:
so pages header in plural and table title in singular or what @dmarcoux ?